api/app/controllers/mno_enterprise/jpi/v1/admin/organizations_controller.rb

```diff
@@ -117,6 +121,78 @@ def invite_member
       @user = user.confirmed? ? invite : user
     end
 
+    def download_batch_example
+      path = File.join(File.dirname(File.expand_path(__FILE__)), '../../../../../assets/batch-example.csv')
```

Comment:
Sorry, did not find a better way to make a reference to a ressource inside the gem.